sidecar/src/setup/unix.rs

```diff
@@ -58,15 +63,15 @@ impl Liaison for SharedDirLiaison {
             // failing to acquire lock
             // means that another process is creating the socket
             Err(err) => {
-                tracing::debug!("failed_locking");
+                debug!("failed_locking");
```

Comment:
Maybe shift this to `warn!`? Seems more important than some of the other debugs like "already_listening" which should be fine.